### PR TITLE
feat: add apy on the mission page 

### PR DESCRIPTION
**Pull request by tcheee**

### CODE REVIEW

File: src/components/Superfest/QuestCard/QuestCard.tsx [hunk 1]
```diff
@@ -110,17 +115,34 @@ export const QuestCard = ({
...
-                  {`+${points}`}
+                  {`${Number(apy).toFixed(1)}%`}
+                </SoraTypography>
+                <XPIconBox marginLeft="4px">
+                    <APYIcon size={20} />
+                </XPIconBox>
+              </XPDisplayBox> : undefined}
+              <XPDisplayBox
+                active={active}
+                bgcolor={!completed ? '#31007A' : '#42B852'}
```
Comment (oktapodia) [inline src/components/Superfest/QuestCard/QuestCard.tsx:137]: Would be good to use the palette

File: src/components/Superfest/SuperfestPage/CTA/MissionCTA.tsx [hunk 2]
```diff
@@ -87,23 +94,46 @@ export const MissionCTA = ({ CTAs }: MissionCtaProps) => {
...
+                    marginTop={{ xs: '16px', md: '0px' }}
                     fontSize={{ xs: '16px', sm: '22px' }}
                     fontWeight={700}
                     marginLeft={'16px'}
                   >
                     {CTA.text ?? 'Go to Protocol Page'}
                   </SoraTypography>
                 </CTAExplanationBox>
+                <FlexCenterRowBox>
+                {CTA.apy ? (
```
Comment (oktapodia) [inline src/components/Superfest/SuperfestPage/CTA/MissionCTA.tsx:106]: Can be replaced by `{CTA.apy && (code)}` for clarity

**Review by oktapodia — CHANGES_REQUESTED** [review]
Couple of comments

File: src/components/Superfest/QuestCard/QuestCard.tsx (hunk 1 above)
Comment (tcheee) [reply src/components/Superfest/QuestCard/QuestCard.tsx:137]: So these colors are not part of the palette as they are one-off colors for a "check" and specific icon. I can include it into the palette but I fear we forget it.

File: src/hooks/useMissionsAPY.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            return {
+              ...CTA,
+              apy: data.apr,
+            };
+          }
+        }
+      }
+    }
+    return CTA;
+  });
```
Comment (admsk2) [inline src/hooks/useMissionsAPY.ts:56]:
```
const CTAsWithAPYs = CTAs.map((CTA: CTALinkInt) => {
  const timestamp = Date.now() / 1000;

  for (const id of ACTIVE_CHAINS) {
    const chainCampaignData = data?.[id]?.[CTA.claimingId];
    
    if (chainCampaignData) {
      for (const [, apyData] of Object.entries(chainCampaignData) as [string, MerklApyRes][]) {
        if (apyData?.apr && apyData.endTimestamp > timestamp) {
          return { ...CTA, apy: apyData.apr };
        }
      }
    }
  }

  return CTA;
});
```

File: src/hooks/useMissionsMaxAPY.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+              data.endTimestamp > timestamp &&
+              data.apr > apy
+            ) {
+              apy = data.apr;
+            }
+          }
+        }
+      }
+    }
+  }
```
Comment (admsk2) [inline src/hooks/useMissionsMaxAPY.ts:59]:
```
if (claimingIds) {
  const timestamp = Date.now() / 1000;
  
  for (const id of claimingIds) {
    for (const chainId of ACTIVE_CHAINS) {
      const chainCampaignData = data?.[chainId]?.[id];
      
      if (chainCampaignData) {
        for (const apyData of Object.values(chainCampaignData) as MerklApyRes[]) {
          if (apyData?.apr && apyData.endTimestamp > timestamp && apyData.apr > apy) {
            apy = apyData.apr;
          }
        }
      }
    }
  }
}
```

**Review by admsk2 — APPROVED** [review]

File: src/hooks/useMissionsAPY.ts (hunk 3 above)
Comment (tcheee) [reply src/hooks/useMissionsAPY.ts:56]: right!

File: src/hooks/useMissionsMaxAPY.ts (hunk 4 above)
Comment (tcheee) [reply src/hooks/useMissionsMaxAPY.ts:59]: thanks

**Review by oktapodia — APPROVED** [review]
LGTM

**Pull request merged**